src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure implements AreaCalculetion {

```

Comment:
The interface name 'AreaCalculetion' should not be a verb. Consider renaming it to something like 'AreaCalculation' to follow Java naming conventions.